misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -114,7 +120,7 @@ class AccessInterceptor private constructor(
       val allowedServices = actionEntries.flatMap { it.services }.toSet()
       val allowedCapabilities = actionEntries.flatMap { it.capabilities }.toSet()
 
-      if (!allowAnyService && allowedServices.isEmpty() && allowedCapabilities.isEmpty()) {
+      if (!allowAnyService && allowedServices.isEmpty() && !allowAnyUser && allowedCapabilities.isEmpty()) {
```

Comment:
maybe we can group the "AllowAny" checks into a common function? Or, group allow any service + allowedServices into a single function. wdyt?